posydon/popsyn/transient_select_funcs.py

```diff
@@ -179,12 +179,12 @@ def BBH_selection_function(history_chunk, oneline_chunk, formation_channels_chun
     df_transients['S2_mass'] = history_chunk[mask]['S2_mass']
     df_transients['S1_spin'] = history_chunk[mask]['S1_spin']
     df_transients['S2_spin'] = history_chunk[mask]['S2_spin']
-    df_transients['S1_spin_orbit_tilt'] = oneline_chunk['S1_spin_orbit_tilt']
-    df_transients['S2_spin_orbit_tilt'] = oneline_chunk['S2_spin_orbit_tilt']
+    df_transients['S1_spin_orbit_tilt'] = oneline_chunk['S1_spin_orbit_tilt_second_SN']
+    df_transients['S2_spin_orbit_tilt'] = oneline_chunk['S2_spin_orbit_tilt_second_SN']
```

Comment:
The naming changed because we split it into `spin_orbit_tilt_first_SN` and `spin_orbit_tilt_second_SN`. Thus, should we include the tilt from the first SN here as well?

Reply:
I kept them different, because for the BBH selection we only care about the `spin_orbit_tilt` at merger, so I kept that one naming convention here.

Reply:
How about calling them then `S1_spin_orbit_tilt_at_merger`?

Reply:
Sounds good! Done!